tests/unit_tests/test_wave_06.py

```diff
@@ -194,16 +202,20 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
+    # - That the results is truthy
+    assert not result
```

Comment:
Looks like you may have accidentally copied a comment that says the opposite of what you want.